Missionframework/scripts/client/build/do_sector_build.sqf

```diff
@@ -54,12 +54,18 @@ if (((_this select 3) select 0) == KP_liberation_small_storage_building) then {
 				_building setVectorUp surfaceNormal position _building;
 			};
 
+            if (build_invalid == 1) then {
+                GRLIB_ui_notif = "";
+            };
```

Comment:
Isn't this nested one level too much?

Reply:
Basically it's an `if` which can be removed, as it doesn't hurt to set it to `""` on each check in the loop. I just redone it like it's in the following from do_build.sqf to have some equality to the other old code (that messy ugly thing in the corner you don't want to look at too long).
https://github.com/KillahPotatoes/KP-Liberation/blob/master/Missionframework/scripts/client/build/do_build.sqf#L219-L252